website/pages/en/quick-start.mdx

```diff
@@ -35,133 +29,158 @@ Using [yarn](https://yarnpkg.com/):
 yarn global add @graphprotocol/graph-cli
 ```
 
-## 3. Initialize your subgraph from existing contract
+### 2. Create your subgraph
 
-Initialize your subgraph from an existing contract by running the initialize command:
+If your contract has events, the `init` command will automatically create a scaffold of a subgraph.
+
+#### Create via Graph CLI
+
+Use the following command to create a subgraph in Subgraph Studio using the CLI:
+
+```sh
+graph init --product subgraph-studio
+```
+
+#### Create via Subgraph Studio
+
+Subgraph Studio has a staging environment where you can create, manage, deploy, and publish subgraphs and API Keys.
```

Comment:
Maybe it's just me, but I find it confusing to talk about Studio's staging environment here. Why not just:
```suggestion
Subgraph Studio lets you create, manage, deploy, and publish subgraphs and API keys.
```

Reply:
Also, some of these actions ("deploy" and "publish") only apply to subgraphs, not API keys, so maybe:
```suggestion
Subgraph Studio lets you create, manage, deploy, and publish subgraphs, as well as create and manage API keys.
```